build.gradle.kts

```diff
@@ -171,6 +171,8 @@ subprojects {
   }
 
   java {
+    sourceCompatibility = JavaVersion.VERSION_16
```

Comment:
why add this?

Reply:
My mistake, I accidentally brought the it after compiling the code. I will remove it

Reply:
Fixed.